File: common-client-api-libs/src/main/java/eu/getsoftware/hotelico/clients/api/clients/infrastructure/menu/dto/MenuItemDTO.java
package eu.getsoftware.hotelico.clients.api.clients.infrastructure.menu.dto;

import eu.getsoftware.hotelico.clients.common.dto.BasicDTO;
import lombok.Getter;
import lombok.With;
import lombok.experimental.SuperBuilder;

import java.sql.Timestamp;

@With
@Getter
@SuperBuilder
public class MenuItemDTO extends BasicDTO{

     private final long orderId = 0L;

     private final int  orderIndex = 0;

     private final long hotelId  = 0L;
     
     private final long cafeId = 0;
     
     private final long senderId  = 0;

     private final boolean delimiter = false;

     private final double price = 0.0;

     private final int amount = 0;

     private final String title = "";

     private final String shortDescription = "";

     private final String description = "";

     private final String pictureUrl = null;

     private final String previewPictureUrl  = null;

     private final Timestamp timestamp = null;
}

File: hotel-client/src/main/kotlin/eu/getsoftware/hotelico/hotel/dto/MenuItemDto.kt
package eu.getsoftware.hotelico.hotel.dto;

import java.sql.Timestamp;

class MenuItemDto: BasicDTO {

     constructor() : super()

     var  orderId = 0L

     var  orderIndex = 0

     var  hotelId: Long? = 0L

     var  cafeId = 0

     var  senderId: Long? = 0

     var  delimiter = false

     var  price = 0.0

     var  amount = 0

     var  title = ""

     var  shortDescription = ""

     var  description = ""

     var  pictureUrl: String? = null

     var  previewPictureUrl: String? = null

     var  timestamp: Timestamp? = null
}
